server/src/internalClusterTest/java/org/elasticsearch/index/IndexRequestBuilderIT.java

```diff
@@ -37,8 +36,10 @@ public void testSetSource() throws InterruptedException, ExecutionException {
                 .setSource(BytesReference.toBytes(new BytesArray("{\"test_field\" : \"foobar\"}")), XContentType.JSON),
             client().prepareIndex("test").setSource(map) };
         indexRandom(true, builders);
-        SearchResponse searchResponse = client().prepareSearch("test").setQuery(QueryBuilders.termQuery("test_field", "foobar")).get();
-        ElasticsearchAssertions.assertHitCount(searchResponse, builders.length);
+        ElasticsearchAssertions.assertHitCount(
```

Comment:
No static import for this one?

Reply:
Didn't have one before and I tried to be as mechnical as possible here :)